package.json

```diff
@@ -59,7 +60,6 @@
     "@wdio/selenium-standalone-service": "^5.0.0",
     "@wdio/sync": "^5.0.0",
```

Comment:
Probably should update these to ^6 sometime soon.

Reply:
> Probably should update these to ^6 sometime soon.

Good idea, I want to take deep look at the dependencies, there are some deprecated and not used ones that should be removed or changed